src/test/kotlin/com/terraformation/backend/nursery/NurserySearchTest.kt

```diff
@@ -0,0 +1,183 @@
+import com.terraformation.backend.search.SearchSortField
+import com.terraformation.backend.search.table.SearchTables
+import io.mockk.every
+import io.mockk.mockk
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneOffset
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class NurserySearchTest : DatabaseTest(), RunsAsUser {
+  override val user = mockUser()
+
+  private val clock: Clock = mockk()
+  private val searchService: SearchService by lazy { SearchService(dslContext) }
+  private val searchTables: SearchTables by lazy { SearchTables(clock) }
+
+  @BeforeEach
+  fun setUp() {
+    every { clock.instant() } returns Instant.EPOCH
+    every { clock.zone } returns ZoneOffset.UTC
+
+    insertUser()
+    insertOrganization(organizationId)
+    insertOrganizationUser(user.userId, organizationId, Role.MANAGER)
+    insertFacility(facilityId, name = "Nursery", type = FacilityType.Nursery)
+  }
+
+  @Test
+  fun `summary tables return correct totals`() {
+    val organizationId2 = OrganizationId(2)
+    val speciesId1 = SpeciesId(1)
+    val speciesId2 = SpeciesId(2)
+    val org2SpeciesId = SpeciesId(3)
+    val facilityId2 = FacilityId(200)
+    val org2FacilityId = FacilityId(300)
+
+    insertOrganization(organizationId2)
+    insertOrganizationUser(user.userId, organizationId2, Role.CONTRIBUTOR)
+
+    insertFacility(facilityId2, name = "Other Nursery", type = FacilityType.Nursery)
+    insertFacility(
+        org2FacilityId,
+        name = "Other Org Nursery",
+        organizationId = organizationId2,
+        type = FacilityType.Nursery)
+
+    insertSpecies(speciesId1)
+    insertSpecies(speciesId2)
+    insertSpecies(org2SpeciesId, organizationId = organizationId2)
+
+    every { user.facilityRoles } returns
+        mapOf(
+            facilityId to Role.MANAGER,
+            facilityId2 to Role.MANAGER,
+            org2FacilityId to Role.CONTRIBUTOR)
+    every { user.organizationRoles } returns
+        mapOf(organizationId to Role.MANAGER, organizationId2 to Role.CONTRIBUTOR)
+
+    insertBatch(
+        facilityId = facilityId,
+        germinatingQuantity = 1,
+        notReadyQuantity = 2,
+        readyQuantity = 4,
+        speciesId = speciesId1)
+    insertBatch(
+        facilityId = facilityId,
+        germinatingQuantity = 8,
+        notReadyQuantity = 16,
+        readyQuantity = 32,
+        speciesId = speciesId1,
+    )
+    insertBatch(
+        facilityId = facilityId2,
+        germinatingQuantity = 64,
+        notReadyQuantity = 128,
+        readyQuantity = 256,
+        speciesId = speciesId1,
+    )
+    insertBatch(
+        facilityId = facilityId,
+        germinatingQuantity = 512,
+        notReadyQuantity = 1024,
+        readyQuantity = 2048,
+        speciesId = speciesId2,
+    )
+    insertBatch(
+        organizationId = organizationId2,
```

Comment:
Correct. Especially since these "tables" are actually the first views we've added to the search implementation, want to make sure there isn't a missing join condition anywhere.